datacatalog/cloud-client/system-test/lookupEntry.test.js

```diff
@@ -18,18 +18,19 @@
 const path = require('path');
 const assert = require('assert');
 const tools = require('@google-cloud/nodejs-repo-tools');
-
 const cwd = path.join(__dirname, '..');
-const projectId = process.env.GCLOUD_PROJECT;
-const datasetId = process.env.GCLOUD_DATASET_ID;
 
 before(tools.checkCredentials);
 
-it('should lookup a dataset entry', async () => {
-  const output = await tools.runAsync(
-    `node lookupEntry.js ${projectId} ${datasetId}`,
-    cwd
-  );
-  const expectedLinkedResource = `//bigquery.googleapis.com/projects/${projectId}/datasets/${datasetId}`;
-  assert.ok(output.includes(expectedLinkedResource));
+describe('lookupEntry lookup', () => {
+  it('should lookup a dataset entry', async () => {
+    const projectId = process.env.GCLOUD_PUBLIC_PROJECT;
```

Comment:
It's not necessary to use environment variables for reading from a public dataset. Please remove these env vars from the kokoro config and use the project / dataset ID directly in this test.

Reply:
done